mne/surface.py

```diff
@@ -1925,7 +1925,7 @@ def warp_montage_volume(montage, base_image, reg_affine, sdr_morph,
     return montage_warped, image_from, image_to
 
 
-_VOXELS_MAX = 100  # define constant to avoid runtime issues
+_VOXELS_MAX = 3 ** 8  # define constant to avoid runtime issues
```

Comment:
I don't understand the logic of 3**8. I would expect maybe something cubed that could be understood as N cubic mm. Would it make sense to use something cubed instead? If not, could you add a comment explaining this choice for the next person who needs to work with this code?

Reply:
You're right that doesn't make sense, I was reversing the exponent. Maybe let's just go back to 1024 or 10 ** 3 = 1000